sql/2019/04_Media/04_09a.sql

```diff
@@ -0,0 +1,17 @@
+/*
+standard SQL
+04_09a
+112 GB
+
+this gets the requests whose servers add a "accept-CH" header
+I'll do another query for the meta tag
+
+*/
+
+
+select url, NET.HOST(url) host, respOtherHeaders
```

Comment:
Is the URL itself useful or do you just want the count of requests?

Are you returning respOtherHeaders for debugging? For the final output this should only return what you expect to use in the written report.

Reply:
im really just looking for the count. I'll fix the query